### DISCUSSION TITLE
The enable-check-update parameter is not used by the RustDesk_default.toml file.

### DISCUSSION THREAD

**Discussion by Vogelbusch** [rustdesk/rustdesk#9451] [General]
### Bug Description

Our problem is to deactivate the automatic version check for all clients.
To distribute settings within our company network, we use the file RustDesk_default.toml.

[options]
view_style = 'adaptive'
enable_file_transfer = 'Y'
show_remote_cursor = 'Y'
enable-check-update = 'N'

The parameter enable-check-update = 'N' is not taken into account by the RustDesk client.

### How to Reproduce

Install RustDesk Client Version 1.3.1
Insert enable-check-update = 'N' to RustDesk_default.toml
Proof Setting "check update" is disabled in RustDesk Client under Settings --> General

### Expected Behavior

Please program RustDesk so that the parameter enable-check-update = 'N' in the file RustDesk_default.toml is taken into account.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

![grafik](https://github.com/user-attachments/assets/6f46f2d5-e8f8-456b-b218-6d56bffbe0ad)

**Comment by rustdesk** [comment]
Here is a tip:

https://github.com/rustdesk/rustdesk/blob/e0095aebdaf7610b164886e505f965be67183ff8/src/flutter_ffi.rs#L1379

**Reply by Vogelbusch** [reply]
Would it please be possible to program the function. We are happy to continue sponsoring the RustDesk project, but we need this function.

**Comment by Vogelbusch** [comment]
![grafik](https://github.com/user-attachments/assets/661c8566-8cda-493e-9194-dea783543bba)
Please help us to deactivate the update request for all clients in our company.
Please program RustDesk so that the option 
enable-check-update = 'N' in the file RustDesk_default.toml is taken into account.
